Adding Deployment Quickstart (#9985)
PrefectHQ/prefect: docs/cloud/cloud-quickstart.md

  boost: 2
---

# Getting Started with Prefect Cloud <span class="badge cloud"></span>

> Get signed in and using Prefect Cloud, including running a flow observed by Prefect Cloud, in just a few steps:

Reviewer: ```suggestion
Get started with Prefect Cloud in just a few steps:
```
